utilities/utils.py

```
@@ -249,45 +249,68 @@ def create_source_provider(
-        )
-
-    if not source_provider_secret:
-        raise ValueError("Failed to create source provider secret")
-
-    # Add copy-offload annotation only when copy-offload is configured
-    provider_annotations = {}
-    if vmware_provider(provider_data=source_provider_data_copy) and has_copyoffload:
-        provider_annotations["forklift.konveyor.io/empty-vddk-init-image"] = "yes"
+        ) as source_provider_secret:
+            if not source_provider_secret:
+                raise ValueError("Failed to create source provider secret")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)